### ISSUE TITLE
Balack bot, App Emulation, KR Solver questions

### ISSUE THREAD

**Issue by heIlo** [kinggreedy/MousehuntUOP#1]
hey there, came here from CnN :P 

so far pretty ok, just have questons about the above 3 functions

balack bot: what does it do? it travels back to cove after washed up is it? but if so why is there an option for shadow trap at JoD? :o 

app emulation: what is this? does it help anything?

KR Solver: what does "this feature is deprecated" mean? is it not safe for use? is there a high chance of ban?

**Comment by heIlo** [comment]
also, the team scoreboard doesn't seem to be working. overall looks great tho, would just like some clarifications regarding the KR solver. I would love to use it but scared ban. :/

**Comment by kinggreedy** [comment]
Balack bot: Yes it travel back to Cove after washed up.In case you don't have any VS cheese left, travel back doesn't do anything, so it will not travel back to Cove and use Shadow trap to farm at JoD instead

App emulation: used to be mobile travel simulator (free travel). But it stopped working because I didn't update it.

KR Solver: It works. The "deprecated feature" is captcha preload (You know your next captcha, so you solve it yourself and the bot will input it when KR is activated). KR solver came out and so it replace the need of preload captcha

Team scoreboard only works when you join tournament

**Comment by heIlo** [comment]
thanks for the clarifications!

i was wondering if there was any script to let me buy/arm new cheese after my current cheese run out?

**Comment by kinggreedy** [comment]
No, because the mechanism is too complex (correct cheese, travel, gold, next condition (example: misting) so this feature is limited to some area, for example, tribal deer dunes farm is capable of buying new resources (cheese).
By the way, if a such condition exists, you probably won't be interested in MH anymore.

**Comment by heIlo** [comment]
oh okay! ya you have a point :P 

Is it possible to make a miniscript for crystal library? Like automatically take the library research assignment everytime I finish one? Thanks!

**Comment by kinggreedy** [comment]
Yes, it's possible to make one
